dev/reference/performance/statistics.md

```diff
@@ -148,22 +238,36 @@ Currently, the `SHOW STATS_BUCKETS` statement returns the following 10 columns:
 
 You can run the `DROP STATS` statement to delete statistics.
 
-Syntax:
+Syntax as follows:
+
+{{< copyable "sql" >}}
 
 ```sql
 DROP STATS TableName
-> The statement deletes statistics of all the tables in `TableName`.
 ```
 
+The statement deletes statistics of all the tables in `TableName`.
+
 ## Import and export statistics
 
 ### Export statistics
 
-The interface to export statistics:
+The interface to export statistics is as follows.
+
+Use this interface to obtain the JSON format statistics of the `${table_name}` table in the `${db_name}` database.
```

Comment:
- Please use this sentence style: `To obtain ..., ...:`.
- Use a list for this section.